.github/workflows/cargo_audit.yml

```diff
@@ -2,16 +2,15 @@ name: Security audit
 on:
   schedule:
     - cron: '0 0 * * *'
+
 jobs:
   audit:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
           target: thumbv7em-none-eabi
-          override: true
```

Comment:
What was the effect of the 2 removed values?

Reply:
setting the toolchain version here overrides the `rust-toolchain` file.
Leaving it unset will try to read the file `rust-toolchain` and abort in case the file doesn't exist.